containers/flatpak/cockpit-client.metainfo.xml.in

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<component type="desktop-application">
+  <id>@FLATPAK_ID@</id>
+
+  <name>Cockpit Client</name>
+  <summary>Connect via ssh to servers running Cockpit</summary>
```

Comment:
That's rather confusing IMHO -- these servers specifically *don't* need to "run" cockpit in the sense of cockpit.{socket,service} or even ws. How about "Cockpit UI for connecting to servers through SSH"?